sdk/program/src/sysvar/slot_hashes.rs

```diff
@@ -62,13 +71,96 @@ impl Sysvar for SlotHashes {
     }
 }
 
+#[derive(Copy, Clone, Pod, Zeroable)]
+#[repr(C)]
+struct PodSlotHash {
+    slot: Slot,
+    hash: Hash,
+}
+
+/// Trait for querying the `SlotHashes` sysvar.
+pub trait SlotHashesSysvar {
```

Comment:
I might be missing some other usage, but why is this a trait? It might be simpler to make it a struct

Reply:
Sure, it could be a struct. I was just keeping things centered on the `SlotHashes` struct, but it would definitely clean up the conflicting method names.